test/integration/error_spam_test.go

```diff
@@ -104,4 +107,103 @@ func TestErrorSpam(t *testing.T) {
 			t.Errorf("missing kubeadm init sub-step %q", step)
 		}
 	}
+
+	logTests := []struct {
+		command          string
+		args             []string
+		runCount         int // number of times to run command
+		expectedLogFiles int // number of logfiles expected after running command runCount times
+	}{
+		{
+			command:          "start",
+			args:             []string{"--dry-run", "--log_dir", os.TempDir()},
+			runCount:         120, // calling this 15 times should create 2 files with 1 greater than 1M
+			expectedLogFiles: 2,
+		},
+		{
+			command:          "status",
+			runCount:         100,
+			expectedLogFiles: 1,
+		}, {
+			command:          "pause",
+			runCount:         5,
+			expectedLogFiles: 1,
+		}, {
+			command:          "unpause",
+			runCount:         1,
+			expectedLogFiles: 1,
+		}, {
+			command:          "stop",
+			runCount:         1,
+			expectedLogFiles: 1,
+		},
+	}
+
+	for _, test := range logTests {
+		t.Run(test.command, func(t *testing.T) {
+			args := []string{test.command, "-p", profile}
+			args = append(args, test.args...)
+			// before starting the test, ensure no other logs from the current command are written
+			logFiles, err := getLogFiles(test.command)
+			if err != nil {
+				t.Errorf("failed to find tmp log files: command %s : %v", test.command, err)
+			}
+			cleanupLogFiles(t, logFiles)
+			// run command runCount times
+			for i := 0; i < test.runCount; i++ {
+				rr, err := Run(t, exec.CommandContext(ctx, Target(), args...))
+				if err != nil {
+					t.Fatalf("%q failed: %v", rr.Command(), err)
+				}
+			}
+			// get log files generated above
+			logFiles, err = getLogFiles(test.command)
+			if err != nil {
+				t.Errorf("failed to find tmp log files: command %s : %v", test.command, err)
+			}
+			// cleanup generated logfiles
+			defer cleanupLogFiles(t, logFiles)
+			// if not the expected number of files, throw err
+			if len(logFiles) != test.expectedLogFiles {
+				t.Errorf("failed to find expected number of log files: cmd %s: expected: %d got %d", test.command, test.expectedLogFiles, len(logFiles))
+			}
+			// if more than 1 logfile is expected, only one file should be less than 1M
+			if test.expectedLogFiles > 1 {
+				foundSmall := false
+				maxSize := 1024 * 1024 // 1M
+				for _, logFile := range logFiles {
+					isSmall := int(logFile.Size()) < maxSize
+					if isSmall && !foundSmall {
+						foundSmall = true
+					} else if isSmall && foundSmall {
+						t.Errorf("expected to find only one file less than 1M: cmd %s:", test.command)
```

Comment:
I think using `1 MB` is a more common abbreviation for megabyte

Reply:
Done